scipy/sparse/linalg/_dsolve/_superlumodule.c

```diff
@@ -263,6 +264,173 @@ static PyObject *Py_gstrf(PyObject * self, PyObject * args,
     return NULL;
 }
 
+static PyObject *Py_gstrs(PyObject * self, PyObject * args,
+                          PyObject * keywds)
+{
+    /* compressed sparse row matrix L */
+    int L_N = 0, L_nnz = 0;
+    PyArrayObject *L_nzvals = NULL, *L_colind = NULL, *L_rowptr = NULL;
+    /* compressed sparse column matrix U */
+    int U_N=0, U_nnz=0;
```

Comment:
```suggestion
    int U_N = 0, U_nnz=0;
```

Reply:
changed in [f530ccf](https://github.com/scipy/scipy/pull/17924/commits/f530ccf711242e1aeaae2af257cecf0e8ff5037e)